src/js/routes/Ballot/Ballot.jsx

```diff
@@ -97,11 +97,11 @@ export default class Ballot extends Component {
     let ballot = this.state.ballot;
     let ballot_props = BallotStore.ballot_properties;
     let ballot_length = 0;
-    if (ballot !== null && ballot.length !== null) {
+    if (typeof ballot !== "undefined" && ballot !== null && ballot.length !== null) {
```

Comment:
I agree about checkout `(ballot && ballot.length)` and also have to wonder why not just `(ballot.length)` because if that is true, then we know there is a `(ballot)`